src/components/z-status-tag/styles.css

```diff
@@ -0,0 +1,132 @@
+z-status-tag {
+  --z-icon-width: 14px;
+  --z-icon-height: 14px;
+
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+
+  height: 22px;
+  border-radius: var(--border-radius);
+  padding-right: 4px;
+  padding-left: 4px;
+  text-transform: uppercase;
+
+  display: inline-flex;
+  align-items: center;
+}
+
+z-status-tag.expandable > z-icon {
+  margin-right: 0;
+  transition: margin-right 1s;
+}
+
+z-status-tag:not(.expandable) > z-icon:not(:last-child),
+z-status-tag.expandable:hover > z-icon:not(:last-child) {
+  margin-right: var(--space-unit);
+}
+
+z-status-tag.expandable > span {
+  max-width: 0;
+  transition: max-width 1s;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+z-status-tag.expandable:hover > span {
+  max-width: 12rem;
```

Comment:
I haven't found any max width indication in the design sheet 🤔

Reply:
Max-width is used for the animation, to show/hide text

Reply:
you can set `100%`

Reply:
Abbiamo provato con 100% ma si spacca l'animazione, succede una cosa brutta: il testo si riduce ma continua ad accupare lo spazio fino a fine animazione

Reply:
forse con `max-width: unset`? in ogni caso settare una max-width fissa non rispetta le specifiche del design